src/content/learn/rendering-lists.md

```diff
@@ -1100,7 +1112,7 @@ This example renders a famous haiku by Tachibana Hokushi, with each line wrapped
 </article>
 ```
 
-A haiku only contains three lines, but your solution should work with any number of lines. Note that `<hr />` elements only appear *between* the `<p>` elements, not in the beginning or the end!
+Haiku sadrži samo tri linije, ali vaše rešenje treba raditi za bilo koji broj linija. Zapazite da se `<hr />` elementi pojavljuju samo *između* `<p>` elemenata, ali ne na početku i kraju!
```

Comment:
I would prefer to leave poem in it's english version, since I didn't found its official serbian version.